src/components/ui/PageHead.tsx

```diff
@@ -210,6 +217,10 @@ export const PageHead = ({
             <link rel="shortcut icon" href="/instill-ai-favicon.ico" />
           </>
         )}
+
+        {pageType === "blog" && (
+          <script type="application/ld+json">{JSON.stringify(jsonLd)}</script>
+        )}
```

Comment:
@iamnamananand996 We need to further check whether JSONLD is present

```suggestion

        {pageType === "blog" && jsonLd && (
          <script type="application/ld+json">{JSON.stringify(jsonLd)}</script>
        )}
```